src/test/java/api/support/fixtures/LostItemFeePoliciesFixture.java

```diff
@@ -23,7 +20,10 @@ public LostItemFeePoliciesFixture(ResourceClient lostItemFeePoliciesClient) {
   }
 
   public IndividualResource facultyStandard() {
+    return facultyStandard(UnaryOperator.identity());
+  }
 
+  public IndividualResource facultyStandard(UnaryOperator<LostItemFeePolicyBuilder> builder) {
```

Comment:
What does this builder represent, additional property configuration for the policy?

Reply:
Yes, it is used for overriding/setting some properties to the default policy.

Reply:
Maybe the parameter could be called something like `additionalConfiguration` to make that intent clearer?

(I've used `additionalProperties` in the past, however I'm not sure I like that any more, because it could be changing the existing state as well)

Reply:
I've extracted policy itself to a separate method, no needs to overload the method.